app/models/user.rb

```diff
@@ -28,6 +28,11 @@ class User < ApplicationRecord
 
   devise :remote_user_authenticatable
 
+  sig { returns(String) }
+  def name
+    self[:name] || 'New SDR User' # return either the name in the database or a default if none exists (i.e. new users)
```

Comment:
Can this be moved into the email, as that is the only place we're expecting this value?

Reply:
Yeah, I considered this, but there are 4 mailers, some with many views, and this change would affect all of them.  It also seems that if we ever show the user's name in the top nav somewhere (as shown in the mockups) this would be desirable there too (@amyehodge ?).  Putting it on the model seemed a like nice clean way to handle it across the board.

Another way if we really do want to isolate to only the emails may be to put a `name` method in `ApplicationMailer` with similar logic and then use that across the mailers instead of using the attribute in the user model.

Reply:
Another way is to make a `UserPresenter` (or `UserDecorator`) object that takes the `User` model as a parameter.  This can have:
```
def name
  model.name || 'New SDR User'
end
```
This means we don't need to have this logic in lots of different places and also keeps it out of the persistence layer.